File: StackMachineEmulator/java/src/tests/UnitTests.java
package tests;

import static org.junit.Assert.*;

import org.junit.Test;

import stackmachineemulator.Solution;

public class UnitTests {

	@Test
	public void testSolution_CanParseReversePolishNotation() {
		String str = "13+62*7+*";
		
        int result = Solution.solution(str);

        assertEquals(76, result);
	}

}

File: StackMachineEmulator/csharp/UnitTests/UnitTests.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;
using StackMachineEmulator;

namespace UnitTests
{
    [TestClass]
    public class UnitTests
    {
        [TestMethod]
        public void Test_Solution_CanParseReversePolishNotation()
        {
            var str = "13+62*7+*";
            var c = new Solution();

            int result = c.solution(str);

            Assert.AreEqual(76, result);
        }
    }
}
